modules/health_assessments/languages/en.json

```diff
@@ -0,0 +1,23 @@
+{
+    "HEALTH_ASSESSMENTS": {
+        "MODULE_DETAILS": {
+            "SINGULAR_NAME": "Health Assessment",
+            "PLURAL_NAME": "Health Assessments"
+        },
+        "FIELDS": {
+            "RECOMMENDATION": "Recommendation",
+            "DESCRIPTION": "Assessment",
+            "SUMMARY": "Summary",
+            "REPORTSANDYAMLFILE": "Reports And YAML file",
```

Comment:
this field name doesn't sound well

Reply:
Please check names i have done some changes